ynput/OpenPype
# Nuke: publish existing frames on farm

## Changelog Description
This PR proposes adding a fourth option in Nuke render publish called "Use Existing Frames - Farm". This would be useful when the farm is busy or when the artist lacks enough farm licenses. Additionally, some artists prefer rendering on the farm but still want to check frames before publishing.

By adding the "Use Existing Frames - Farm" option, artists will have more flexibility and control over their render publishing process. This enhancement will streamline the workflow and improve efficiency for Nuke users.

## Additional info
Validate Existing Frames on farm needed to be refactored so it is including also the logic where no dependent jobs are needed.

This PR has been originally @fabiaserra contribution https://github.com/ynput/OpenPype/pull/5196. Thank you very much!

## To do:
- [x] fix `submit_nuke_deadline` so it submit only baking part
- [x] test new `collect_write` extensively with slates

## Testing notes:
1. In Nuke test task create render instance and render locally
2. set render instance to `Use existing frames - farm`
3. publish and see that one OpenPype job is on farm
4. make sure all reviewable publishing was completed as expected
